bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/JBLangVMErrors.java

```diff
@@ -0,0 +1,39 @@
+/*
+*   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.ballerinalang.jvm;
+
+import org.ballerinalang.jvm.values.ErrorValue;
+import org.ballerinalang.jvm.values.MapValue;
+
+/**
+ * Util Class for handling Error in Ballerina VM.
+ *
+ * @since 0.995.0
+ */
+public class JBLangVMErrors {
```

Comment:
we dont have to call this BVM errors as it will be not there. Shall we just use some simple names?

Reply:
+1. Yes name bit complex. will fix